frontends/home/Home.module.scss

```diff
@@ -10,5 +10,9 @@
     background: rgb(204, 204, 204);
     border-radius: 1rem;
   }
+}
 
-}
\ No newline at end of file
+.label-buttons {
+  margin-right: 0.5rem;
+  width: fit-content;
```

Comment:
it is not working
<img width="1440" alt="Screenshot 2023-04-04 at 22 32 23" src="https://user-images.githubusercontent.com/55176773/229871981-0ce09570-1898-4b45-84af-0e65003ea845.png">